idioms-json-2.1-custom/coa_ind_malicious_file.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "bundle--769a7cac-c00a-4ef9-ac4e-c3493acd5856",
+    "objects": [
+        {
+            "administrative_area": "US-DC",
+            "country": "US",
+            "created": "9999-12-31T00:00:00.000Z",
+            "id": "location--271e4cbc-b78f-43da-ab90-6f90cafa605e",
+            "modified": "9999-12-31T00:00:00.000Z",
+            "spec_version": "2.1",
+            "type": "location"
+        },
+        {
+            "created": "9999-12-31T00:00:00.000Z",
+            "id": "relationship--d4d13323-7ed7-4fe5-86ed-294770d40004",
+            "modified": "9999-12-31T00:00:00.000Z",
+            "relationship_type": "located-at",
+            "source_ref": "identity--007a335e-26b2-4fec-af20-9e61bb1ffc19",
+            "spec_version": "2.1",
+            "target_ref": "location--271e4cbc-b78f-43da-ab90-6f90cafa605e",
+            "type": "relationship"
+        },
+        {
+            "created": "9999-12-31T00:00:00.000Z",
+            "id": "identity--007a335e-26b2-4fec-af20-9e61bb1ffc19",
+            "identity_class": "organization",
+            "modified": "9999-12-31T00:00:00.000Z",
+            "name": "ACME",
+            "object_marking_refs": [
+                "marking-definition--e13519a9-84c7-44b2-8dff-929238089152"
+            ],
+            "sectors": [
+                "technology"
+            ],
+            "spec_version": "2.1",
+            "type": "identity"
+        },
+        {
+            "created": "9999-12-31T00:00:00.000Z",
+            "created_by_ref": "identity--007a335e-26b2-4fec-af20-9e61bb1ffc19",
+            "definition": {
+                "consent": "everyone",
+                "is_cisa_proprietary": "false",
+                "is_proprietary": "false",
+                "tlp": "white"
+            },
+            "definition_type": "ais",
+            "id": "marking-definition--e13519a9-84c7-44b2-8dff-929238089152",
+            "spec_version": "2.1",
+            "type": "marking-definition"
+        },
+        {
+            "created": "2016-03-29T20:52:52.716Z",
+            "description": "Send Device for forensic analysis",
+            "id": "course-of-action--e9f31688-d744-48f4-9320-e6a9bd136f44",
+            "modified": "2016-03-29T20:52:52.716Z",
+            "name": "Bricked Computer",
+            "object_marking_refs": [
+                "marking-definition--e13519a9-84c7-44b2-8dff-929238089152"
+            ],
+            "spec_version": "2.1",
+            "type": "course-of-action"
+        },
+        {
+            "created": "2016-03-29T20:52:52.716Z",
+            "id": "indicator--0c277b59-f5a9-4530-8861-82588719ec0c",
+            "kill_chain_phases": [
+                {
+                    "kill_chain_name": "LM Cyber Kill Chain",
+                    "phase_name": "Actions on Objectives"
```

Comment:
LM Cyber Kill Chain has special requirements:

2.1 https://docs.oasis-open.org/cti/stix/v2.1/cs01/stix-v2.1-cs01.html#_i4tjv75ce50h
2.0 http://docs.oasis-open.org/cti/stix/v2.0/cs01/part1-stix-core/stix-v2.0-cs01-part1-stix-core.html#_Toc496709267
(It can be fixed on a different PR - perhaps just tracking it as an issue for now)

Reply:
See issue #226